orchestration/dagster_orchestration/hca_orchestration/resources/data_repo.py

```diff
@@ -24,7 +24,7 @@ def base_jade_data_repo_client(init_context: InitResourceContext):
 
 
 @configured(base_jade_data_repo_client)
-def jade_data_repo_client(config):
+def jade_data_repo_client(_config):
```

Comment:
❓ What does the _ prefix mean? Unused?

Reply:
yup, generally i prepend an underscore if the variable is required in the signature but not used in the body